tbx/people/tests/test_author.py

```diff
@@ -0,0 +1,122 @@
+from django.test import TestCase
+
+from tbx.core.factories import HomePageFactory
+from tbx.images.factories import CustomImageFactory
+from tbx.people.models import Author, PersonIndexPage, PersonPage
```

Comment:
**Suggestion**: Factories for these might make your life easier in future, too. See how much more verbose the PersonPage creation steps are in the tests below.